services/ui-src/src/components/layout/Section.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+// components
+import { Box, Flex, Heading, Text } from "@chakra-ui/react";
+import { WidgetContainer } from "components/widgets/WidgetContainer";
```

Comment:
Suggestion: Could just reference this from components directly because of the way we're exporting it.
```suggestion
import { WidgetContainer } from "components";
```